### DISCUSSION TITLE
NO .EXE

### DISCUSSION THREAD

**Discussion by BlackPhantomm** [Xaymar/obs-stream-effects#918] [Q&A] [answered]
IVE LOOKED EVERYWHERE ON BOTH 0/11/1 AND THE BETA RELEASE, AND THERE IS NO .EXE RUN FILE IN THE FILES ANYWHERE

**Comment by Xaymar** [comment, marked as answer]
Extend the Assets dropdown. There's plenty of exes.

**Comment by BlackPhantomm** [comment]
there isnt an assets folder

**Reply by Xaymar** [reply]
The installation guide links to the latest Production release, as well as all Releases: https://github.com/Xaymar/obs-StreamFX/wiki/Installation . Simply pick whichever works best for your setup, read all the release notes, and scroll down to download it.
